event.json

```diff
@@ -0,0 +1,14 @@
+{
+  "pull_request": {
+    "number": 1,
+    "title": "Test PR",
```

Comment:
The title of the pull request should be more descriptive. "Test PR" does not convey the purpose or content of the pull request effectively.